en/contributing-git.md

```diff
@@ -0,0 +1,345 @@
+Title: Working with Git and GitHub
+TODO:  Review commented out section: "Additional resources"
+table_of_contents: True
+
+
+# Working with Git and GitHub
+
+This page provides the essentials for working with Git and GitHub with the goal
+of contributing specifically to MAAS documentation. See the upstream projects
+for definitive documentation: [Git docs][git-docs] and
+[GitHub docs][github-docs].
+
+
+## Initial setup
+
+Several actions are required before you can begin. These only need to be done
+once.
+
+- GitHub account
+- User environment
+- Fork the MAAS docs repository on GitHub
+- Clone your MAAS docs fork locally
+- Add the upstream remote
+
+### GitHub account
+
+A [GitHub account][github-account] will be required. Sign up now!
+
+Verify your email address by responding to the email notification you will
+receive.
+
+The remainder of this page will refer to your GitHub username as
+**$GH_USERNAME** (e.g. 'johnsmith').
+
+### User environment
+
+Configure some user environment essentials.
+
+Set your user name and email address:
+
+`git config --global user.name "$FIRST_NAME $LAST_NAME"`
```

Comment:
I forgot to add that I could have used it in this particular instance but then I would break consistency later ('General workflow'). What do you think?